analyses/fusion-summary/01-fusion-summary.R

```diff
@@ -0,0 +1,167 @@
+#' @description  Generate fusion files specifically for consumption by molecular subtyping analyses
+#' @author Daniel Miller <millerd15@@email.chop.edu> (D3b)
+#' @note Date: January 2020
+
+suppressPackageStartupMessages(library("dplyr"))
+suppressPackageStartupMessages(library("data.table"))
+suppressPackageStartupMessages(library("optparse"))
+
+#' **Filters**
+#' 
+#' *Demographic Filers*
+#' 1: Used to filter short_histology to get Ependymoma tumors 
+ependSH = "Ependymoma"
+#' 2: Used for Embryonal tumors (short_histology)
+#'  not from ATRT or MB (broad_histology)
+embryBH = "Embryonal tumor"
+embrySH = c("ATRT", "Medulloblastoma")
+#' 
+#' *Fusions Filters*
+#' 1: Exact match a list of fusions common in Ependymoma tumors
+ependFuses = c(
```

Comment:
@jharenza does the order of the partners matters for these? My read of #245 was yes

Reply:
yes, correct, these are the fusions in the orientation that have been reported